packages/libs/react-ui/src/components/Notification/NotificationButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { SystemIcon } from '../Icons';
+
+import { colorVariants, iconContainerClass } from './NotificationButton.css';
+
+import React, { FC } from 'react';
+
+export interface INotificationButtonProps {
+  icon: typeof SystemIcon[keyof typeof SystemIcon];
+  color: keyof typeof colorVariants;
+  children: React.ReactNode;
+}
+export const NotificationButton: FC<INotificationButtonProps> = ({
+  icon,
+  color,
+  children,
+}) => {
+  const Icon = icon;
+  return (
+    <button className={colorVariants[color]}>
+      {children}
+      <div className={iconContainerClass}>
```

Comment:
When wrapping svgs, you should use inline elements like spans to make the html more semantically correct. Divs are meant to be used as block elements to group more items together, but this icon is meant to be inline with the text so a span makes more sense in this case and in most cases